### DISCUSSION TITLE
automate this in the wrapper / stdlib so devs dont have to define every time (whats with the globalThis hate ?!)

### DISCUSSION THREAD

**Discussion by ohitstom** [Delusoire/bespoke#8] [Ideas]
![image](https://github.com/Delusoire/bespoke/assets/22730962/aaa16bf7-2a24-4bc6-8658-f4c2b79db3f3)

**Comment by Delusoire** [comment]
Unfortunately that's not going to happen :(

**Comment by ohitstom** [comment]
> Unfortunately that's not going to happen :(

any reason why? I dread having to call like 15 platform methods and defining it at the top of all my extensions :/

**Comment by JustSxm** [comment]
It just makes sense to import only what you require, and not everthing

**Reply by ohitstom** [reply]
> It just makes sense to import only what you require, and not everthing

disagree.

spicetifys API should be globally exposed and we shouldn't be forcing users to define every single platform method just to use it.

this will break 100% of extensions too whereas if this is fixed it'll only be a fraction of extensions that need updating for v3 at all - most of my extensions use purely platform methods and if I wasn't actively maintaining them this choice will break them indefinitely.
